sunpy/net/dataretriever/sources/swap.py

```diff
@@ -0,0 +1,107 @@
+''' SWAPClient definition '''
+
+__author__ = "Sudarshan Konge"
+__email__ = "sudk1896@gmail.com"
+
+import datetime
+import urllib2
+
+
+from astropy import units as u
+from bs4 import BeautifulSoup
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+
+from sunpy.time import TimeRange
+
+__all__ = ['SWAPClient']
+
+class SWAPClient(GenericClient):
+    
+    """ Returns a list of URLS to Proba2 SWAP files corresponding to value of input timerange.
+    URL source: http://proba2.oma.be/swap/data/bsd/
```

Comment:
The hyperlink should be in rst hyperlink format http://www.sphinx-doc.org/en/stable/rest.html#external-links